db/migrate/20170519135150_create_lessons.rb

```diff
@@ -0,0 +1,16 @@
+class CreateLessons < ActiveRecord::Migration[5.0]
+  def change
+    create_table :lessons do |t|
+      t.text :byline
+      t.string :lesson_type
```

Comment:
Totally optional: What if we use the postgres enum type instead for `lesson_type`? Rails supports it with `enum lesson_type: { type0: 0, type1: 1 }` in the model.